css/main.css

```diff
@@ -495,6 +495,14 @@ input[name="data.money"] {
   background-repeat: no-repeat;
   background-position: 2px 4px;
 }
+
+input[name="totalwealth"] {
+  background-image: url(/systems/ptu/css/images/Pokémon_Dollar_sign.svg);
+  padding-left: 20px;
+  background-size: 12px 16px;
+  background-repeat: no-repeat;
+  background-position: 2px 4px;
+}
```

Comment:
Do not directly change the CSS, instead put changes in the .LESS files otherwise they **will be voided** the next time someone makes any LESS changes.

Reply:
applied changes to less, these have been automatically applied to the css files